README.md

```diff
@@ -54,11 +56,9 @@ Contents are added frequently! ⚡
 ---
 
 ## Cyber Security
-| Article Title | Author | Article Link |
-| ------------- | ------ | ------------ |
-|               |        |              |
-
----
+| Week Number | Article Title | Author | Article Link |
```

Comment:
I think we should have it monthly instead, or we should make it open. Remember, the goal is to encourage everyone, and now that we have a repo, we can accept more. What do you think? @segunajibola

Reply:
I don't understand this. The week number will be for the week number the article was published. week 1 - 52.

Reply:
Yeah, I'll do that now